fiftyone/utils/glm_ocr.py

```
@@ -0,0 +1,194 @@
+        img = np.clip(img, 0, 255)
+    img = img.astype(np.uint8)
+    if img.ndim == 2:
+        img = np.stack([img] * 3, axis=-1)
+    if img.shape[2] == 1:
+        img = np.repeat(img, 3, axis=2)
+    elif img.shape[2] == 4:
+        img = img[:, :, :3]
+    return PILImage.fromarray(img)
+
+
+def _select_dtype(device):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_select_dtype`

(ANN202)